  INFO root [2019-09-14 09:49:41,405] - 
  INFO root [2019-09-14 09:49:41,405] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 09:49:41,406] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 09:49:41,406] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 09:49:41,407] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 09:49:41,421] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:64344 )
  INFO root [2019-09-14 09:49:41,422] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:64344, SOMAXCONN=5.
  INFO root [2019-09-14 09:49:41,422] - server-configs:
  INFO root [2019-09-14 09:49:41,422] - 	internalTcpAddr : 192.168.56.1:64344
  INFO root [2019-09-14 09:49:41,422] - 	componentID : 6000
  WARN root [2019-09-14 09:49:41,423] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 09:49:42,928] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13536 ----
  INFO root [2019-09-14 09:49:42,929] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 09:49:42,929] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 09:49:42,929] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 09:49:43,348] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 09:49:43,348] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 09:49:50,490] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 09:49:50,551] - Components::findComponents: found logger, addr:192.168.56.1:64341
  INFO root [2019-09-14 09:49:50,551] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 09:49:50,551] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 09:49:50,553] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 09:49:50,557] - Components::findComponents: found dbmgr, addr:192.168.56.1:64342
  INFO root [2019-09-14 09:49:50,557] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 09:49:50,557] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 09:49:50,561] - Components::findComponents: found logger, addr:192.168.56.1:64341
  INFO root [2019-09-14 09:49:50,565] - Components::findComponents: found dbmgr, addr:192.168.56.1:64342
  WARN root [2019-09-14 09:49:51,565] - Forwarding logs to logger[192.168.56.1:64341/0/0/0]...
 ERROR root [2019-09-14 09:52:42,462] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520045462776, lastReceivedTime=520042269480, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-14 09:52:42,467] - The following logs sent to logger failed:
  WARN root [2019-09-14 09:52:42,467] - ==> [2019-09-14 09:52:42 462] TCPPacketReceiver::processPendingEvents(192.168.56.1:64341): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 09:52:42,467] - ==> [2019-09-14 09:52:42 463] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520045462776, lastReceivedTime=520042269480, inactivityExceptionPeriod=134887325)
  WARN root [2019-09-14 09:52:42,559] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64342): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 09:52:42,561] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520045683408, lastReceivedTime=520042911158, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-14 09:52:42,565] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 09:52:42,566] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 09:52:42,566] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 09:52:42,566] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 09:52:44,147] - 
  INFO root [2019-09-14 09:52:44,154] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 09:52:44,155] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 09:52:44,155] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 09:52:44,155] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 09:52:44,227] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:64380 )
  INFO root [2019-09-14 09:52:44,227] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:64380, SOMAXCONN=5.
  INFO root [2019-09-14 09:52:44,227] - server-configs:
  INFO root [2019-09-14 09:52:44,227] - 	internalTcpAddr : 192.168.56.1:64380
  INFO root [2019-09-14 09:52:44,227] - 	componentID : 6000
  WARN root [2019-09-14 09:52:44,236] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 09:52:45,750] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 12232 ----
  INFO root [2019-09-14 09:52:45,751] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 09:52:45,751] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 09:52:45,751] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 09:52:46,154] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 09:52:46,156] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 09:52:53,273] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 09:52:53,378] - Components::findComponents: found logger, addr:192.168.56.1:64376
  INFO root [2019-09-14 09:52:53,378] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 09:52:53,379] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 09:52:53,380] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 09:52:53,384] - Components::findComponents: found logger, addr:192.168.56.1:64376
  INFO root [2019-09-14 09:52:53,387] - Components::findComponents: found dbmgr, addr:192.168.56.1:64377
  INFO root [2019-09-14 09:52:53,387] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 09:52:53,387] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 09:52:53,488] - Components::findComponents: found dbmgr, addr:192.168.56.1:64377
  WARN root [2019-09-14 09:52:54,489] - Forwarding logs to logger[192.168.56.1:64376/0/0/0]...
 ERROR root [2019-09-14 09:55:02,036] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520359502448, lastReceivedTime=520319151956, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-14 09:55:02,039] - The following logs sent to logger failed:
  WARN root [2019-09-14 09:55:02,039] - ==> [2019-09-14 09:55:02 035] TCPPacketReceiver::processPendingEvents(192.168.56.1:64376): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 09:55:02,040] - ==> [2019-09-14 09:55:02 036] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520359502448, lastReceivedTime=520319151956, inactivityExceptionPeriod=134887325)
  WARN root [2019-09-14 09:55:02,138] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64377): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 09:55:02,145] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=520359748776, lastReceivedTime=520319243661, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-14 09:55:02,146] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 09:55:02,147] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 09:55:02,147] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 09:55:02,147] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 10:00:43,059] - 
  INFO root [2019-09-14 10:00:43,069] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 10:00:43,070] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 10:00:43,070] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 10:00:43,071] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 10:00:43,138] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:64419 )
  INFO root [2019-09-14 10:00:43,138] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:64419, SOMAXCONN=5.
  INFO root [2019-09-14 10:00:43,138] - server-configs:
  INFO root [2019-09-14 10:00:43,138] - 	internalTcpAddr : 192.168.56.1:64419
  INFO root [2019-09-14 10:00:43,138] - 	componentID : 6000
  WARN root [2019-09-14 10:00:43,150] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 10:00:44,656] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 16728 ----
  INFO root [2019-09-14 10:00:44,657] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 10:00:44,658] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 10:00:44,660] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 10:00:45,131] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 10:00:45,131] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 10:00:52,282] - Components::findComponents: find logger(1)...
  WARN root [2019-09-14 10:00:52,612] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20693, errorno=10048
  INFO root [2019-09-14 10:00:53,622] - Components::findComponents: find logger(2)...
  WARN root [2019-09-14 10:00:53,953] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20082, errorno=10048
  INFO root [2019-09-14 10:00:54,962] - Components::findComponents: find logger(3)...
  INFO root [2019-09-14 10:00:54,985] - Components::findComponents: found logger, addr:192.168.56.1:64418
  INFO root [2019-09-14 10:00:54,986] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 10:00:54,986] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 10:00:54,988] - Components::findComponents: find dbmgr(4)...
  INFO root [2019-09-14 10:00:55,094] - Components::findComponents: found dbmgr, addr:192.168.56.1:64417
  INFO root [2019-09-14 10:00:55,094] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 10:00:55,095] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 10:00:55,099] - Components::findComponents: found dbmgr, addr:192.168.56.1:64417
  WARN root [2019-09-14 10:00:56,101] - Forwarding logs to logger[192.168.56.1:64418/0/0/0]...
 ERROR root [2019-09-14 10:05:07,945] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=521722799771, lastReceivedTime=521666968483, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 10:05:07,951] - The following logs sent to logger failed:
  WARN root [2019-09-14 10:05:07,951] - ==> [2019-09-14 10:05:07 945] TCPPacketReceiver::processPendingEvents(192.168.56.1:64418): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 10:05:07,951] - ==> [2019-09-14 10:05:07 946] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=521722799771, lastReceivedTime=521666968483, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 10:05:08,033] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64417): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 10:05:08,035] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=521723002954, lastReceivedTime=521666915515, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 10:05:08,041] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 10:05:08,043] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 10:05:08,043] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 10:05:08,043] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 10:07:02,959] - 
  INFO root [2019-09-14 10:07:02,962] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 10:07:02,963] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 10:07:02,963] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 10:07:02,963] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 10:07:02,987] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:64457 )
  INFO root [2019-09-14 10:07:02,987] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:64457, SOMAXCONN=5.
  INFO root [2019-09-14 10:07:02,987] - server-configs:
  INFO root [2019-09-14 10:07:02,987] - 	internalTcpAddr : 192.168.56.1:64457
  INFO root [2019-09-14 10:07:02,987] - 	componentID : 6000
  WARN root [2019-09-14 10:07:02,989] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 10:07:04,495] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13812 ----
  INFO root [2019-09-14 10:07:04,496] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 10:07:04,496] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 10:07:04,496] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 10:07:04,882] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 10:07:04,886] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 10:07:11,998] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 10:07:12,101] - Components::findComponents: found logger, addr:192.168.56.1:64460
  INFO root [2019-09-14 10:07:12,101] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 10:07:12,102] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 10:07:12,103] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 10:07:12,106] - Components::findComponents: found dbmgr, addr:192.168.56.1:64462
  INFO root [2019-09-14 10:07:12,106] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 10:07:12,107] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 10:07:12,207] - Components::findComponents: found dbmgr, addr:192.168.56.1:64462
  WARN root [2019-09-14 10:07:13,209] - Forwarding logs to logger[192.168.56.1:64460/0/0/0]...
 ERROR root [2019-09-14 10:14:27,404] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522981581002, lastReceivedTime=522926690681, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 10:14:27,408] - The following logs sent to logger failed:
  WARN root [2019-09-14 10:14:27,408] - ==> [2019-09-14 10:14:27 402] TCPPacketReceiver::processPendingEvents(192.168.56.1:64460): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 10:14:27,409] - ==> [2019-09-14 10:14:27 404] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522981581002, lastReceivedTime=522926690681, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 10:14:27,497] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64462): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 10:14:27,501] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=522981800758, lastReceivedTime=522926816006, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 10:14:27,504] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 10:14:27,510] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 10:14:27,510] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 10:14:27,511] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 10:14:29,028] - 
  INFO root [2019-09-14 10:14:29,030] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 10:14:29,031] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 10:14:29,032] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 10:14:29,032] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 10:14:29,054] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:64498 )
  INFO root [2019-09-14 10:14:29,054] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:64498, SOMAXCONN=5.
  INFO root [2019-09-14 10:14:29,055] - server-configs:
  INFO root [2019-09-14 10:14:29,055] - 	internalTcpAddr : 192.168.56.1:64498
  INFO root [2019-09-14 10:14:29,055] - 	componentID : 6000
  WARN root [2019-09-14 10:14:29,062] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 10:14:30,575] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13080 ----
  INFO root [2019-09-14 10:14:30,576] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 10:14:30,576] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 10:14:30,576] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 10:14:30,977] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 10:14:30,981] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 10:14:38,178] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 10:14:38,183] - Components::findComponents: found logger, addr:192.168.56.1:64497
  INFO root [2019-09-14 10:14:38,183] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 10:14:38,183] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 10:14:38,185] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 10:14:38,285] - Components::findComponents: found logger, addr:192.168.56.1:64497
  INFO root [2019-09-14 10:14:38,288] - Components::findComponents: found dbmgr, addr:192.168.56.1:64500
  INFO root [2019-09-14 10:14:38,288] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 10:14:38,288] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 10:14:38,289] - Components::findComponents: found dbmgr, addr:192.168.56.1:64500
  WARN root [2019-09-14 10:14:39,290] - Forwarding logs to logger[192.168.56.1:64497/0/0/0]...
 ERROR root [2019-09-14 10:18:54,506] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=523582561938, lastReceivedTime=523525341388, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 10:18:54,512] - The following logs sent to logger failed:
  WARN root [2019-09-14 10:18:54,512] - ==> [2019-09-14 10:18:54 504] TCPPacketReceiver::processPendingEvents(192.168.56.1:64497): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 10:18:54,512] - ==> [2019-09-14 10:18:54 506] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=523582561938, lastReceivedTime=523525341388, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 10:18:54,593] - TCPPacketReceiver::processPendingEvents(192.168.56.1:64500): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 10:18:54,595] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=523582762554, lastReceivedTime=523525432507, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 10:18:54,599] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 10:18:54,599] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 10:18:54,599] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 10:18:54,599] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:04:38,382] - 
  INFO root [2019-09-14 15:04:38,415] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:04:38,416] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:04:38,416] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:04:38,417] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:04:38,432] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51062 )
  INFO root [2019-09-14 15:04:38,432] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51062, SOMAXCONN=5.
  INFO root [2019-09-14 15:04:38,432] - server-configs:
  INFO root [2019-09-14 15:04:38,432] - 	internalTcpAddr : 192.168.56.1:51062
  INFO root [2019-09-14 15:04:38,433] - 	componentID : 6000
  WARN root [2019-09-14 15:04:38,433] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 15:04:38,803] - Components::findLogger: found logger, addr:192.168.56.1:51060
  INFO root [2019-09-14 15:04:38,803] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:04:38,803] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:04:38,815] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13872 ----
  INFO root [2019-09-14 15:04:38,821] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:04:38,825] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:04:38,826] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:04:39,285] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:04:39,287] - Components::process(): Request for the process of identity...
  WARN root [2019-09-14 15:04:45,439] - Forwarding logs to logger[192.168.56.1:51060/0/0/0]...
 ERROR root [2019-09-14 15:08:24,959] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=30040472673, lastReceivedTime=30003123739, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:08:24,968] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:08:24,968] - ==> [2019-09-14 15:08:24 952] TCPPacketReceiver::processPendingEvents(192.168.56.1:51060): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:08:24,968] - ==> [2019-09-14 15:08:24 960] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=30040472673, lastReceivedTime=30003123739, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-14 15:08:25,084] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51061): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:08:25,084] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=30040754536, lastReceivedTime=30003209553, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:08:25,087] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:08:25,089] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:08:25,089] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:08:25,090] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:11:38,935] - 
  INFO root [2019-09-14 15:11:38,938] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:11:38,939] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:11:38,939] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:11:38,941] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:11:38,984] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51196 )
  INFO root [2019-09-14 15:11:38,985] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51196, SOMAXCONN=5.
  INFO root [2019-09-14 15:11:38,985] - server-configs:
  INFO root [2019-09-14 15:11:38,985] - 	internalTcpAddr : 192.168.56.1:51196
  INFO root [2019-09-14 15:11:38,985] - 	componentID : 6000
  WARN root [2019-09-14 15:11:38,986] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 15:11:40,490] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 5724 ----
  INFO root [2019-09-14 15:11:40,491] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:11:40,492] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:11:40,492] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:11:40,899] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:11:40,899] - Components::process(): Request for the process of identity...
  WARN root [2019-09-14 15:11:47,072] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51211): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-14 15:11:48,073] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:11:48,102] - Components::findComponents: found logger, addr:192.168.56.1:51190
  INFO root [2019-09-14 15:11:48,102] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:11:48,103] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:11:48,104] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:11:48,207] - Components::findComponents: found dbmgr, addr:192.168.56.1:51192
  INFO root [2019-09-14 15:11:48,207] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:11:48,207] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:11:48,207] - Components::findComponents: found dbmgr, addr:192.168.56.1:51192
  WARN root [2019-09-14 15:11:49,209] - Forwarding logs to logger[192.168.56.1:51190/0/0/0]...
 ERROR root [2019-09-14 15:13:58,939] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=30791930689, lastReceivedTime=30746878986, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 15:13:58,946] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:13:58,946] - ==> [2019-09-14 15:13:58 938] TCPPacketReceiver::processPendingEvents(192.168.56.1:51190): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:13:58,947] - ==> [2019-09-14 15:13:58 942] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=30791930689, lastReceivedTime=30746878986, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 15:13:59,046] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51192): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:13:59,046] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=30792170660, lastReceivedTime=30746984350, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 15:13:59,051] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:13:59,051] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:13:59,051] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:13:59,051] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:16:28,201] - 
  INFO root [2019-09-14 15:16:28,202] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:16:28,203] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:16:28,203] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:16:28,203] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:16:28,221] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51287 )
  INFO root [2019-09-14 15:16:28,221] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51287, SOMAXCONN=5.
  INFO root [2019-09-14 15:16:28,221] - server-configs:
  INFO root [2019-09-14 15:16:28,221] - 	internalTcpAddr : 192.168.56.1:51287
  INFO root [2019-09-14 15:16:28,221] - 	componentID : 6000
  WARN root [2019-09-14 15:16:28,222] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 15:16:29,730] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13800 ----
  INFO root [2019-09-14 15:16:29,732] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:16:29,732] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:16:29,733] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:16:30,145] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:16:30,145] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 15:16:37,294] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:16:37,328] - Components::findComponents: found logger, addr:192.168.56.1:51288
  INFO root [2019-09-14 15:16:37,328] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:16:37,329] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:16:37,330] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:16:37,433] - Components::findComponents: found dbmgr, addr:192.168.56.1:51290
  INFO root [2019-09-14 15:16:37,433] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:16:37,433] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:16:37,443] - Components::findComponents: found dbmgr, addr:192.168.56.1:51290
  WARN root [2019-09-14 15:16:38,443] - Forwarding logs to logger[192.168.56.1:51288/0/0/0]...
 ERROR root [2019-09-14 15:19:55,213] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=31593545806, lastReceivedTime=31532840438, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:19:55,219] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:19:55,220] - ==> [2019-09-14 15:19:55 209] TCPPacketReceiver::processPendingEvents(192.168.56.1:51288): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:19:55,220] - ==> [2019-09-14 15:19:55 214] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=31593545806, lastReceivedTime=31532840438, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-14 15:19:55,340] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51290): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:19:55,345] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=31593842838, lastReceivedTime=31532923892, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:19:55,354] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:19:55,368] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:19:55,369] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:19:55,370] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:20:11,923] - 
  INFO root [2019-09-14 15:20:11,934] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:20:11,935] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:20:11,935] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:20:11,935] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:20:11,995] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51330 )
  INFO root [2019-09-14 15:20:11,995] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51330, SOMAXCONN=5.
  INFO root [2019-09-14 15:20:11,996] - server-configs:
  INFO root [2019-09-14 15:20:11,996] - 	internalTcpAddr : 192.168.56.1:51330
  INFO root [2019-09-14 15:20:11,996] - 	componentID : 6000
  WARN root [2019-09-14 15:20:11,997] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 15:20:13,501] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 7560 ----
  INFO root [2019-09-14 15:20:13,502] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:20:13,502] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:20:13,502] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:20:13,968] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:20:13,968] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 15:20:21,144] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:20:21,166] - Components::findComponents: found logger, addr:192.168.56.1:51327
  INFO root [2019-09-14 15:20:21,167] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:20:21,167] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:20:21,168] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:20:21,173] - Components::findComponents: found logger, addr:192.168.56.1:51327
  INFO root [2019-09-14 15:20:21,176] - Components::findComponents: found dbmgr, addr:192.168.56.1:51332
  INFO root [2019-09-14 15:20:21,177] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:20:21,177] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:20:21,278] - Components::findComponents: found dbmgr, addr:192.168.56.1:51332
  WARN root [2019-09-14 15:20:22,279] - Forwarding logs to logger[192.168.56.1:51327/0/0/0]...
 ERROR root [2019-09-14 15:22:46,984] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=31980031923, lastReceivedTime=31968624037, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:22:46,986] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:22:46,987] - ==> [2019-09-14 15:22:46 984] TCPPacketReceiver::processPendingEvents(192.168.56.1:51327): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:22:46,987] - ==> [2019-09-14 15:22:46 985] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=31980031923, lastReceivedTime=31968624037, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-14 15:22:47,082] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51332): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:22:47,088] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=31980263421, lastReceivedTime=31968839967, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:22:47,096] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:22:47,098] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:22:47,100] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:22:47,100] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:22:48,778] - 
  INFO root [2019-09-14 15:22:48,786] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:22:48,788] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:22:48,788] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:22:48,788] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:22:48,829] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51367 )
  INFO root [2019-09-14 15:22:48,829] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51367, SOMAXCONN=5.
  INFO root [2019-09-14 15:22:48,829] - server-configs:
  INFO root [2019-09-14 15:22:48,830] - 	internalTcpAddr : 192.168.56.1:51367
  INFO root [2019-09-14 15:22:48,830] - 	componentID : 6000
  WARN root [2019-09-14 15:22:48,831] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-14 15:22:49,198] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20367, errorno=10048
  INFO root [2019-09-14 15:22:49,202] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 14256 ----
  INFO root [2019-09-14 15:22:49,203] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:22:49,203] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:22:49,203] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:22:49,619] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:22:49,620] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 15:22:56,785] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:22:56,880] - Components::findComponents: found logger, addr:192.168.56.1:51368
  INFO root [2019-09-14 15:22:56,880] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:22:56,880] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:22:56,882] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:22:56,984] - Components::findComponents: found dbmgr, addr:192.168.56.1:51369
  INFO root [2019-09-14 15:22:56,984] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:22:56,985] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:22:56,985] - Components::findComponents: found dbmgr, addr:192.168.56.1:51369
  WARN root [2019-09-14 15:22:57,986] - Forwarding logs to logger[192.168.56.1:51368/0/0/0]...
 ERROR root [2019-09-14 15:24:32,437] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=32217302153, lastReceivedTime=32186701599, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 15:24:32,440] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:24:32,440] - ==> [2019-09-14 15:24:32 434] TCPPacketReceiver::processPendingEvents(192.168.56.1:51368): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:24:32,441] - ==> [2019-09-14 15:24:32 439] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=32217302153, lastReceivedTime=32186701599, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 15:24:32,537] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51369): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:24:32,540] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=32217532353, lastReceivedTime=32186724182, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 15:24:32,552] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:24:32,567] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:24:32,567] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:24:32,567] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:29:31,335] - 
  INFO root [2019-09-14 15:29:31,350] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:29:31,352] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:29:31,352] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:29:31,352] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:29:31,438] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51408 )
  INFO root [2019-09-14 15:29:31,438] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51408, SOMAXCONN=5.
  INFO root [2019-09-14 15:29:31,438] - server-configs:
  INFO root [2019-09-14 15:29:31,439] - 	internalTcpAddr : 192.168.56.1:51408
  INFO root [2019-09-14 15:29:31,439] - 	componentID : 6000
  WARN root [2019-09-14 15:29:31,447] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  WARN root [2019-09-14 15:29:31,793] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20451, errorno=10048
  INFO root [2019-09-14 15:29:31,798] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 5488 ----
  INFO root [2019-09-14 15:29:31,798] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:29:31,798] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:29:31,799] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:29:32,262] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:29:32,263] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 15:29:39,460] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:29:39,472] - Components::findComponents: found logger, addr:192.168.56.1:51404
  INFO root [2019-09-14 15:29:39,472] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:29:39,472] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:29:39,473] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:29:39,576] - Components::findComponents: found dbmgr, addr:192.168.56.1:51411
  INFO root [2019-09-14 15:29:39,576] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:29:39,577] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:29:39,577] - Components::findComponents: found dbmgr, addr:192.168.56.1:51411
  WARN root [2019-09-14 15:29:40,577] - Forwarding logs to logger[192.168.56.1:51404/0/0/0]...
 ERROR root [2019-09-14 15:31:24,775] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=33145061027, lastReceivedTime=33092326497, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:31:24,782] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:31:24,783] - ==> [2019-09-14 15:31:24 775] TCPPacketReceiver::processPendingEvents(192.168.56.1:51404): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:31:24,783] - ==> [2019-09-14 15:31:24 776] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=33145061027, lastReceivedTime=33092326497, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-14 15:31:24,883] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51411): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:31:24,888] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=33145315316, lastReceivedTime=33092600276, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:31:24,908] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:31:24,918] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:31:24,918] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:31:24,918] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:34:29,679] - 
  INFO root [2019-09-14 15:34:29,685] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:34:29,686] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:34:29,686] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:34:29,686] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:34:29,719] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51465 )
  INFO root [2019-09-14 15:34:29,719] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51465, SOMAXCONN=5.
  INFO root [2019-09-14 15:34:29,719] - server-configs:
  INFO root [2019-09-14 15:34:29,719] - 	internalTcpAddr : 192.168.56.1:51465
  INFO root [2019-09-14 15:34:29,720] - 	componentID : 6000
  WARN root [2019-09-14 15:34:29,734] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 15:34:31,238] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15232 ----
  INFO root [2019-09-14 15:34:31,239] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:34:31,239] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:34:31,240] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:34:31,640] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:34:31,641] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 15:34:38,762] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:34:38,807] - Components::findComponents: found logger, addr:192.168.56.1:51466
  INFO root [2019-09-14 15:34:38,808] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:34:38,808] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:34:38,809] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:34:38,912] - Components::findComponents: found dbmgr, addr:192.168.56.1:51471
  INFO root [2019-09-14 15:34:38,912] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:34:38,912] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:34:38,913] - Components::findComponents: found dbmgr, addr:192.168.56.1:51471
  WARN root [2019-09-14 15:34:39,914] - Forwarding logs to logger[192.168.56.1:51466/0/0/0]...
 ERROR root [2019-09-14 15:36:56,955] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=33892466515, lastReceivedTime=33831253704, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:36:56,957] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:36:56,958] - ==> [2019-09-14 15:36:56 950] TCPPacketReceiver::processPendingEvents(192.168.56.1:51466): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:36:56,958] - ==> [2019-09-14 15:36:56 956] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=33892466515, lastReceivedTime=33831253704, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-14 15:36:57,066] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51471): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:36:57,070] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=33892726456, lastReceivedTime=33831345432, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 15:36:57,095] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:36:57,097] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:36:57,098] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:36:57,099] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:40:16,172] - 
  INFO root [2019-09-14 15:40:16,174] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:40:16,175] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:40:16,175] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:40:16,175] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:40:16,200] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51509 )
  INFO root [2019-09-14 15:40:16,200] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51509, SOMAXCONN=5.
  INFO root [2019-09-14 15:40:16,200] - server-configs:
  INFO root [2019-09-14 15:40:16,200] - 	internalTcpAddr : 192.168.56.1:51509
  INFO root [2019-09-14 15:40:16,201] - 	componentID : 6000
  WARN root [2019-09-14 15:40:16,202] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 15:40:17,708] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 6848 ----
  INFO root [2019-09-14 15:40:17,709] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:40:17,709] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:40:17,709] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:40:18,149] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:40:18,150] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 15:40:25,302] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:40:25,371] - Components::findComponents: found logger, addr:192.168.56.1:51504
  INFO root [2019-09-14 15:40:25,371] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:40:25,371] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:40:25,373] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:40:25,377] - Components::findComponents: found logger, addr:192.168.56.1:51504
  INFO root [2019-09-14 15:40:25,379] - Components::findComponents: found dbmgr, addr:192.168.56.1:51507
  INFO root [2019-09-14 15:40:25,380] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:40:25,380] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:40:25,482] - Components::findComponents: found dbmgr, addr:192.168.56.1:51507
  WARN root [2019-09-14 15:40:26,482] - Forwarding logs to logger[192.168.56.1:51504/0/0/0]...
 ERROR root [2019-09-14 15:43:14,747] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=34742500014, lastReceivedTime=34678090244, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 15:43:14,754] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:43:14,754] - ==> [2019-09-14 15:43:14 743] TCPPacketReceiver::processPendingEvents(192.168.56.1:51504): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:43:14,754] - ==> [2019-09-14 15:43:14 748] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=34742500014, lastReceivedTime=34678090244, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 15:43:14,842] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51507): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:43:14,848] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=34742726682, lastReceivedTime=34678191944, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 15:43:14,858] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:43:14,864] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:43:14,864] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:43:14,864] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:43:16,559] - 
  INFO root [2019-09-14 15:43:16,559] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:43:16,560] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:43:16,560] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:43:16,561] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:43:16,576] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51548 )
  INFO root [2019-09-14 15:43:16,576] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51548, SOMAXCONN=5.
  INFO root [2019-09-14 15:43:16,576] - server-configs:
  INFO root [2019-09-14 15:43:16,576] - 	internalTcpAddr : 192.168.56.1:51548
  INFO root [2019-09-14 15:43:16,576] - 	componentID : 6000
  WARN root [2019-09-14 15:43:16,577] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 15:43:18,083] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 14108 ----
  INFO root [2019-09-14 15:43:18,083] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:43:18,084] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:43:18,084] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:43:18,489] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:43:18,491] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 15:43:25,646] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:43:25,723] - Components::findComponents: found logger, addr:192.168.56.1:51546
  INFO root [2019-09-14 15:43:25,723] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:43:25,723] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:43:25,725] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:43:25,729] - Components::findComponents: found logger, addr:192.168.56.1:51546
  INFO root [2019-09-14 15:43:25,731] - Components::findComponents: found dbmgr, addr:192.168.56.1:51547
  INFO root [2019-09-14 15:43:25,731] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:43:25,732] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:43:25,833] - Components::findComponents: found dbmgr, addr:192.168.56.1:51547
  WARN root [2019-09-14 15:43:26,834] - Forwarding logs to logger[192.168.56.1:51546/0/0/0]...
 ERROR root [2019-09-14 15:47:40,994] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=35341554616, lastReceivedTime=35286496892, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-14 15:47:41,005] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:47:41,006] - ==> [2019-09-14 15:47:40 994] TCPPacketReceiver::processPendingEvents(192.168.56.1:51546): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:47:41,006] - ==> [2019-09-14 15:47:40 995] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=35341554616, lastReceivedTime=35286496892, inactivityExceptionPeriod=134887628)
  WARN root [2019-09-14 15:47:41,123] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51547): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:47:41,143] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=35341890730, lastReceivedTime=35286532721, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-14 15:47:41,156] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:47:41,159] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:47:41,159] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:47:41,160] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:47:46,390] - 
  INFO root [2019-09-14 15:47:46,391] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:47:46,392] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:47:46,392] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:47:46,392] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:47:46,411] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51585 )
  INFO root [2019-09-14 15:47:46,411] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51585, SOMAXCONN=5.
  INFO root [2019-09-14 15:47:46,412] - server-configs:
  INFO root [2019-09-14 15:47:46,412] - 	internalTcpAddr : 192.168.56.1:51585
  INFO root [2019-09-14 15:47:46,412] - 	componentID : 6000
  WARN root [2019-09-14 15:47:46,416] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 15:47:47,938] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 12676 ----
  INFO root [2019-09-14 15:47:47,938] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:47:47,939] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:47:47,939] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:47:48,384] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:47:48,385] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 15:47:55,540] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:47:55,573] - Components::findComponents: found logger, addr:192.168.56.1:51583
  INFO root [2019-09-14 15:47:55,574] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:47:55,574] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:47:55,578] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:47:55,683] - Components::findComponents: found dbmgr, addr:192.168.56.1:51586
  INFO root [2019-09-14 15:47:55,683] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:47:55,683] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:47:55,692] - Components::findComponents: found dbmgr, addr:192.168.56.1:51586
  WARN root [2019-09-14 15:47:56,693] - Forwarding logs to logger[192.168.56.1:51583/0/0/0]...
 ERROR root [2019-09-14 15:51:01,900] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=35793592407, lastReceivedTime=35758619588, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-14 15:51:01,902] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:51:01,903] - ==> [2019-09-14 15:51:01 898] TCPPacketReceiver::processPendingEvents(192.168.56.1:51583): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:51:01,903] - ==> [2019-09-14 15:51:01 900] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=35793592407, lastReceivedTime=35758619588, inactivityExceptionPeriod=134887325)
  WARN root [2019-09-14 15:51:02,018] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51586): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:51:02,024] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=35793872358, lastReceivedTime=35758847251, inactivityExceptionPeriod=134887325)
  INFO root [2019-09-14 15:51:02,052] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:51:02,058] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:51:02,058] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:51:02,058] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 15:54:13,336] - 
  INFO root [2019-09-14 15:54:13,343] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 15:54:13,344] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 15:54:13,344] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 15:54:13,344] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 15:54:13,389] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51627 )
  INFO root [2019-09-14 15:54:13,390] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51627, SOMAXCONN=5.
  INFO root [2019-09-14 15:54:13,390] - server-configs:
  INFO root [2019-09-14 15:54:13,390] - 	internalTcpAddr : 192.168.56.1:51627
  INFO root [2019-09-14 15:54:13,390] - 	componentID : 6000
  WARN root [2019-09-14 15:54:13,391] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 15:54:14,904] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 3408 ----
  INFO root [2019-09-14 15:54:14,904] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 15:54:14,905] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 15:54:14,905] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 15:54:15,307] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 15:54:15,310] - Components::process(): Request for the process of identity...
  WARN root [2019-09-14 15:54:16,070] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20450, errorno=10048
  INFO root [2019-09-14 15:54:23,266] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 15:54:23,368] - Components::findComponents: found logger, addr:192.168.56.1:51626
  INFO root [2019-09-14 15:54:23,368] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:54:23,368] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:54:23,370] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 15:54:23,373] - Components::findComponents: found dbmgr, addr:192.168.56.1:51623
  INFO root [2019-09-14 15:54:23,373] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 15:54:23,373] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 15:54:23,375] - Components::findComponents: found logger, addr:192.168.56.1:51626
  INFO root [2019-09-14 15:54:23,379] - Components::findComponents: found dbmgr, addr:192.168.56.1:51623
  WARN root [2019-09-14 15:54:24,379] - Forwarding logs to logger[192.168.56.1:51626/0/0/0]...
 ERROR root [2019-09-14 15:58:58,771] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=36866553631, lastReceivedTime=36831898502, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 15:58:58,774] - The following logs sent to logger failed:
  WARN root [2019-09-14 15:58:58,774] - ==> [2019-09-14 15:58:58 764] TCPPacketReceiver::processPendingEvents(192.168.56.1:51626): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:58:58,775] - ==> [2019-09-14 15:58:58 772] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=36866553631, lastReceivedTime=36831898502, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 15:58:58,888] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51623): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 15:58:58,891] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=36866823704, lastReceivedTime=36831856872, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 15:58:58,909] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 15:58:58,915] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 15:58:58,916] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 15:58:58,917] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 16:00:37,147] - 
  INFO root [2019-09-14 16:00:37,152] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 16:00:37,154] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 16:00:37,154] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 16:00:37,154] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 16:00:37,323] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51672 )
  INFO root [2019-09-14 16:00:37,323] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51672, SOMAXCONN=5.
  INFO root [2019-09-14 16:00:37,324] - server-configs:
  INFO root [2019-09-14 16:00:37,324] - 	internalTcpAddr : 192.168.56.1:51672
  INFO root [2019-09-14 16:00:37,324] - 	componentID : 6000
  WARN root [2019-09-14 16:00:37,326] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 16:00:38,863] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 12840 ----
  INFO root [2019-09-14 16:00:38,867] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 16:00:38,867] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 16:00:38,868] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 16:00:39,402] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 16:00:39,403] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 16:00:46,569] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 16:00:46,641] - Components::findComponents: found logger, addr:192.168.56.1:51665
  INFO root [2019-09-14 16:00:46,641] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:00:46,641] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:00:46,643] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 16:00:46,746] - Components::findComponents: found dbmgr, addr:192.168.56.1:51671
  INFO root [2019-09-14 16:00:46,746] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:00:46,746] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:00:46,746] - Components::findComponents: found dbmgr, addr:192.168.56.1:51671
  WARN root [2019-09-14 16:00:47,747] - Forwarding logs to logger[192.168.56.1:51665/0/0/0]...
 ERROR root [2019-09-14 16:03:00,766] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=37411041641, lastReceivedTime=37357510563, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 16:03:00,776] - The following logs sent to logger failed:
  WARN root [2019-09-14 16:03:00,776] - ==> [2019-09-14 16:03:00 761] TCPPacketReceiver::processPendingEvents(192.168.56.1:51665): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:03:00,776] - ==> [2019-09-14 16:03:00 767] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=37411041641, lastReceivedTime=37357510563, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 16:03:00,886] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51671): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:03:00,893] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=37411330483, lastReceivedTime=37358105469, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 16:03:00,909] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 16:03:00,914] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 16:03:00,915] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 16:03:00,915] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 16:03:02,835] - 
  INFO root [2019-09-14 16:03:02,868] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 16:03:02,870] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 16:03:02,870] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 16:03:02,870] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 16:03:02,930] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51707 )
  INFO root [2019-09-14 16:03:02,931] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51707, SOMAXCONN=5.
  INFO root [2019-09-14 16:03:02,931] - server-configs:
  INFO root [2019-09-14 16:03:02,935] - 	internalTcpAddr : 192.168.56.1:51707
  INFO root [2019-09-14 16:03:02,936] - 	componentID : 6000
  WARN root [2019-09-14 16:03:02,952] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 16:03:04,468] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 13984 ----
  INFO root [2019-09-14 16:03:04,469] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 16:03:04,470] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 16:03:04,470] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 16:03:04,987] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 16:03:04,989] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 16:03:12,178] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 16:03:12,285] - Components::findComponents: found logger, addr:192.168.56.1:51709
  INFO root [2019-09-14 16:03:12,286] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:03:12,286] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:03:12,288] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 16:03:12,293] - Components::findComponents: found logger, addr:192.168.56.1:51709
  INFO root [2019-09-14 16:03:12,296] - Components::findComponents: found dbmgr, addr:192.168.56.1:51712
  INFO root [2019-09-14 16:03:12,297] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:03:12,297] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:03:12,400] - Components::findComponents: found dbmgr, addr:192.168.56.1:51712
  WARN root [2019-09-14 16:03:13,401] - Forwarding logs to logger[192.168.56.1:51709/0/0/0]...
 ERROR root [2019-09-14 16:06:45,796] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=37917360681, lastReceivedTime=37888476743, inactivityExceptionPeriod=134887197)
  INFO root [2019-09-14 16:06:45,804] - The following logs sent to logger failed:
  WARN root [2019-09-14 16:06:45,805] - ==> [2019-09-14 16:06:45 795] TCPPacketReceiver::processPendingEvents(192.168.56.1:51709): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:06:45,805] - ==> [2019-09-14 16:06:45 797] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=37917360681, lastReceivedTime=37888476743, inactivityExceptionPeriod=134887197)
  WARN root [2019-09-14 16:06:45,902] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51712): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:06:45,904] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=37917604687, lastReceivedTime=37888575215, inactivityExceptionPeriod=134887197)
  INFO root [2019-09-14 16:06:45,911] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 16:06:45,914] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 16:06:45,915] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 16:06:45,915] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 16:06:47,719] - 
  INFO root [2019-09-14 16:06:47,736] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 16:06:47,738] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 16:06:47,738] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 16:06:47,738] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 16:06:47,770] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51747 )
  INFO root [2019-09-14 16:06:47,771] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51747, SOMAXCONN=5.
  INFO root [2019-09-14 16:06:47,771] - server-configs:
  INFO root [2019-09-14 16:06:47,771] - 	internalTcpAddr : 192.168.56.1:51747
  INFO root [2019-09-14 16:06:47,771] - 	componentID : 6000
  WARN root [2019-09-14 16:06:47,774] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 16:06:49,289] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15876 ----
  INFO root [2019-09-14 16:06:49,289] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 16:06:49,290] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 16:06:49,290] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 16:06:49,721] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 16:06:49,723] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 16:06:56,891] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 16:06:56,938] - Components::findComponents: found logger, addr:192.168.56.1:51744
  INFO root [2019-09-14 16:06:56,939] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:06:56,939] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:06:56,940] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 16:06:56,945] - Components::findComponents: found logger, addr:192.168.56.1:51744
  INFO root [2019-09-14 16:06:56,948] - Components::findComponents: found dbmgr, addr:192.168.56.1:51746
  INFO root [2019-09-14 16:06:56,949] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:06:56,949] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:06:57,050] - Components::findComponents: found dbmgr, addr:192.168.56.1:51746
  WARN root [2019-09-14 16:06:58,051] - Forwarding logs to logger[192.168.56.1:51744/0/0/0]...
 ERROR root [2019-09-14 16:10:16,201] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=38390773051, lastReceivedTime=38326854006, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 16:10:16,209] - The following logs sent to logger failed:
  WARN root [2019-09-14 16:10:16,209] - ==> [2019-09-14 16:10:16 198] TCPPacketReceiver::processPendingEvents(192.168.56.1:51744): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:10:16,209] - ==> [2019-09-14 16:10:16 203] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=38390773051, lastReceivedTime=38326854006, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-14 16:10:16,331] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51746): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:10:16,341] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=38391087648, lastReceivedTime=38326730113, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 16:10:16,354] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 16:10:16,358] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 16:10:16,359] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 16:10:16,359] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-14 16:10:16,660] - ThreadPool::destroy(): waiting for thread(2)[0x84aa80,0x84ac20,], try=1
  INFO root [2019-09-14 16:46:30,242] - 
  INFO root [2019-09-14 16:46:30,263] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 16:46:30,264] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 16:46:30,264] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 16:46:30,264] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 16:46:30,308] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51802 )
  INFO root [2019-09-14 16:46:30,308] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51802, SOMAXCONN=5.
  INFO root [2019-09-14 16:46:30,309] - server-configs:
  INFO root [2019-09-14 16:46:30,309] - 	internalTcpAddr : 192.168.56.1:51802
  INFO root [2019-09-14 16:46:30,309] - 	componentID : 6000
  WARN root [2019-09-14 16:46:30,309] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 16:46:31,813] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15356 ----
  INFO root [2019-09-14 16:46:31,814] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 16:46:31,814] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 16:46:31,814] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 16:46:32,319] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 16:46:32,328] - Components::process(): Request for the process of identity...
  WARN root [2019-09-14 16:46:32,842] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20005, errorno=10048
  WARN root [2019-09-14 16:46:33,198] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20742, errorno=10048
  INFO root [2019-09-14 16:46:40,332] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 16:46:40,425] - Components::findComponents: found logger, addr:192.168.56.1:51797
  INFO root [2019-09-14 16:46:40,426] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:46:40,426] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:46:40,427] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 16:46:40,530] - Components::findComponents: found dbmgr, addr:192.168.56.1:51799
  INFO root [2019-09-14 16:46:40,531] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:46:40,531] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:46:40,531] - Components::findComponents: found dbmgr, addr:192.168.56.1:51799
  WARN root [2019-09-14 16:46:41,531] - Forwarding logs to logger[192.168.56.1:51797/0/0/0]...
 ERROR root [2019-09-14 16:47:58,734] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=43481472801, lastReceivedTime=43417383549, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 16:47:58,735] - The following logs sent to logger failed:
  WARN root [2019-09-14 16:47:58,736] - ==> [2019-09-14 16:47:58 730] TCPPacketReceiver::processPendingEvents(192.168.56.1:51797): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:47:58,736] - ==> [2019-09-14 16:47:58 735] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=43481472801, lastReceivedTime=43417383549, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 16:47:58,846] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51799): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:47:58,851] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=43481736669, lastReceivedTime=43417445914, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 16:47:58,867] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 16:47:58,868] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 16:47:58,868] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 16:47:58,868] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 16:48:00,561] - 
  INFO root [2019-09-14 16:48:00,574] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 16:48:00,575] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 16:48:00,575] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 16:48:00,575] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 16:48:00,628] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51836 )
  INFO root [2019-09-14 16:48:00,629] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51836, SOMAXCONN=5.
  INFO root [2019-09-14 16:48:00,629] - server-configs:
  INFO root [2019-09-14 16:48:00,629] - 	internalTcpAddr : 192.168.56.1:51836
  INFO root [2019-09-14 16:48:00,629] - 	componentID : 6000
  WARN root [2019-09-14 16:48:00,630] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 16:48:02,142] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 7712 ----
  INFO root [2019-09-14 16:48:02,155] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 16:48:02,176] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 16:48:02,176] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 16:48:02,697] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 16:48:02,699] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 16:48:09,882] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 16:48:09,927] - Components::findComponents: found logger, addr:192.168.56.1:51831
  INFO root [2019-09-14 16:48:09,927] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:48:09,927] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:48:09,929] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 16:48:10,033] - Components::findComponents: found dbmgr, addr:192.168.56.1:51832
  INFO root [2019-09-14 16:48:10,034] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:48:10,034] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:48:10,034] - Components::findComponents: found dbmgr, addr:192.168.56.1:51832
  WARN root [2019-09-14 16:48:11,035] - Forwarding logs to logger[192.168.56.1:51831/0/0/0]...
 ERROR root [2019-09-14 16:52:19,727] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=44068706475, lastReceivedTime=44025400946, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 16:52:19,733] - The following logs sent to logger failed:
  WARN root [2019-09-14 16:52:19,733] - ==> [2019-09-14 16:52:19 726] TCPPacketReceiver::processPendingEvents(192.168.56.1:51831): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:52:19,733] - ==> [2019-09-14 16:52:19 727] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=44068706475, lastReceivedTime=44025400946, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-14 16:52:19,824] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51832): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:52:19,830] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=44068941849, lastReceivedTime=44025508233, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 16:52:19,835] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 16:52:19,844] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 16:52:19,844] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 16:52:19,845] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 16:52:21,561] - 
  INFO root [2019-09-14 16:52:21,566] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 16:52:21,567] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 16:52:21,567] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 16:52:21,567] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 16:52:21,592] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51873 )
  INFO root [2019-09-14 16:52:21,593] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51873, SOMAXCONN=5.
  INFO root [2019-09-14 16:52:21,593] - server-configs:
  INFO root [2019-09-14 16:52:21,593] - 	internalTcpAddr : 192.168.56.1:51873
  INFO root [2019-09-14 16:52:21,593] - 	componentID : 6000
  WARN root [2019-09-14 16:52:21,594] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 16:52:23,099] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 2976 ----
  INFO root [2019-09-14 16:52:23,100] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 16:52:23,100] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 16:52:23,100] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 16:52:23,499] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 16:52:23,501] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 16:52:30,662] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 16:52:30,722] - Components::findComponents: found logger, addr:192.168.56.1:51871
  INFO root [2019-09-14 16:52:30,722] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:52:30,723] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:52:30,724] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 16:52:30,728] - Components::findComponents: found dbmgr, addr:192.168.56.1:51869
  INFO root [2019-09-14 16:52:30,728] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:52:30,729] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:52:30,832] - Components::findComponents: found dbmgr, addr:192.168.56.1:51869
  WARN root [2019-09-14 16:52:31,832] - Forwarding logs to logger[192.168.56.1:51871/0/0/0]...
 ERROR root [2019-09-14 16:55:22,851] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=44480735845, lastReceivedTime=44477723690, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-14 16:55:22,853] - The following logs sent to logger failed:
  WARN root [2019-09-14 16:55:22,853] - ==> [2019-09-14 16:55:22 850] TCPPacketReceiver::processPendingEvents(192.168.56.1:51871): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:55:22,854] - ==> [2019-09-14 16:55:22 851] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=44480735845, lastReceivedTime=44477723690, inactivityExceptionPeriod=134887628)
  WARN root [2019-09-14 16:55:22,968] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51869): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:55:22,971] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=44481005561, lastReceivedTime=44477685454, inactivityExceptionPeriod=134887628)
  INFO root [2019-09-14 16:55:22,981] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 16:55:22,984] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 16:55:22,984] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 16:55:22,985] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 16:57:03,937] - 
  INFO root [2019-09-14 16:57:03,960] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 16:57:03,962] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 16:57:03,963] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 16:57:03,963] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 16:57:04,012] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51918 )
  INFO root [2019-09-14 16:57:04,013] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51918, SOMAXCONN=5.
  INFO root [2019-09-14 16:57:04,013] - server-configs:
  INFO root [2019-09-14 16:57:04,013] - 	internalTcpAddr : 192.168.56.1:51918
  INFO root [2019-09-14 16:57:04,013] - 	componentID : 6000
  WARN root [2019-09-14 16:57:04,014] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 16:57:05,521] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 16240 ----
  INFO root [2019-09-14 16:57:05,521] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 16:57:05,522] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 16:57:05,522] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 16:57:06,016] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 16:57:06,017] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 16:57:13,166] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 16:57:13,256] - Components::findComponents: found logger, addr:192.168.56.1:51912
  INFO root [2019-09-14 16:57:13,257] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:57:13,257] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:57:13,261] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 16:57:13,365] - Components::findComponents: found dbmgr, addr:192.168.56.1:51914
  INFO root [2019-09-14 16:57:13,365] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 16:57:13,365] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 16:57:13,366] - Components::findComponents: found dbmgr, addr:192.168.56.1:51914
  WARN root [2019-09-14 16:57:14,367] - Forwarding logs to logger[192.168.56.1:51912/0/0/0]...
 ERROR root [2019-09-14 16:58:29,266] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=44900172286, lastReceivedTime=44842787854, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 16:58:29,269] - The following logs sent to logger failed:
  WARN root [2019-09-14 16:58:29,269] - ==> [2019-09-14 16:58:29 266] TCPPacketReceiver::processPendingEvents(192.168.56.1:51912): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:58:29,269] - ==> [2019-09-14 16:58:29 267] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=44900172286, lastReceivedTime=44842787854, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-14 16:58:29,369] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51914): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 16:58:29,375] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=44900415917, lastReceivedTime=44843133455, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 16:58:29,382] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 16:58:29,383] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 16:58:29,383] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 16:58:29,383] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 17:01:33,323] - 
  INFO root [2019-09-14 17:01:33,326] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 17:01:33,328] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 17:01:33,328] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 17:01:33,328] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 17:01:33,378] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51954 )
  INFO root [2019-09-14 17:01:33,379] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51954, SOMAXCONN=5.
  INFO root [2019-09-14 17:01:33,379] - server-configs:
  INFO root [2019-09-14 17:01:33,379] - 	internalTcpAddr : 192.168.56.1:51954
  INFO root [2019-09-14 17:01:33,379] - 	componentID : 6000
  WARN root [2019-09-14 17:01:33,383] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 17:01:34,893] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 12608 ----
  INFO root [2019-09-14 17:01:34,894] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 17:01:34,894] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 17:01:34,894] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 17:01:35,342] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 17:01:35,343] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 17:01:42,476] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 17:01:42,497] - Components::findComponents: found logger, addr:192.168.56.1:51949
  INFO root [2019-09-14 17:01:42,497] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 17:01:42,497] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 17:01:42,499] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 17:01:42,602] - Components::findComponents: found dbmgr, addr:192.168.56.1:51955
  INFO root [2019-09-14 17:01:42,602] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 17:01:42,602] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 17:01:42,602] - Components::findComponents: found dbmgr, addr:192.168.56.1:51955
  WARN root [2019-09-14 17:01:43,603] - Forwarding logs to logger[192.168.56.1:51949/0/0/0]...
 ERROR root [2019-09-14 17:03:29,805] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=45576383320, lastReceivedTime=45516722422, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 17:03:29,813] - The following logs sent to logger failed:
  WARN root [2019-09-14 17:03:29,813] - ==> [2019-09-14 17:03:29 802] TCPPacketReceiver::processPendingEvents(192.168.56.1:51949): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:03:29,814] - ==> [2019-09-14 17:03:29 805] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=45576383320, lastReceivedTime=45516722422, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 17:03:29,913] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51955): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:03:29,915] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=45576632659, lastReceivedTime=45516953241, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 17:03:29,918] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 17:03:29,919] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 17:03:29,919] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 17:03:29,920] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 17:03:31,684] - 
  INFO root [2019-09-14 17:03:31,691] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 17:03:31,692] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 17:03:31,693] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 17:03:31,693] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 17:03:31,715] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:51988 )
  INFO root [2019-09-14 17:03:31,715] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:51988, SOMAXCONN=5.
  INFO root [2019-09-14 17:03:31,715] - server-configs:
  INFO root [2019-09-14 17:03:31,715] - 	internalTcpAddr : 192.168.56.1:51988
  INFO root [2019-09-14 17:03:31,715] - 	componentID : 6000
  WARN root [2019-09-14 17:03:31,719] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 17:03:33,229] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15044 ----
  INFO root [2019-09-14 17:03:33,229] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 17:03:33,229] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 17:03:33,230] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 17:03:33,626] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 17:03:33,627] - Components::process(): Request for the process of identity...
  WARN root [2019-09-14 17:03:33,967] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20795, errorno=10048
  INFO root [2019-09-14 17:03:41,151] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 17:03:41,192] - Components::findComponents: found logger, addr:192.168.56.1:51994
  INFO root [2019-09-14 17:03:41,193] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 17:03:41,193] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 17:03:41,194] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 17:03:41,297] - Components::findComponents: found dbmgr, addr:192.168.56.1:51987
  INFO root [2019-09-14 17:03:41,297] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 17:03:41,297] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 17:03:41,297] - Components::findComponents: found dbmgr, addr:192.168.56.1:51987
  WARN root [2019-09-14 17:03:42,298] - Forwarding logs to logger[192.168.56.1:51994/0/0/0]...
 ERROR root [2019-09-14 17:05:44,937] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=45880430531, lastReceivedTime=45850871173, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 17:05:44,941] - The following logs sent to logger failed:
  WARN root [2019-09-14 17:05:44,941] - ==> [2019-09-14 17:05:44 934] TCPPacketReceiver::processPendingEvents(192.168.56.1:51994): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:05:44,941] - ==> [2019-09-14 17:05:44 937] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=45880430531, lastReceivedTime=45850871173, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 17:05:45,036] - TCPPacketReceiver::processPendingEvents(192.168.56.1:51987): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:05:45,036] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=45880654289, lastReceivedTime=45850702256, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 17:05:45,040] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 17:05:45,040] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 17:05:45,040] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 17:05:45,041] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 17:05:46,795] - 
  INFO root [2019-09-14 17:05:46,812] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 17:05:46,813] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 17:05:46,813] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 17:05:46,813] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 17:05:46,936] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:52032 )
  INFO root [2019-09-14 17:05:46,936] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:52032, SOMAXCONN=5.
  INFO root [2019-09-14 17:05:46,936] - server-configs:
  INFO root [2019-09-14 17:05:46,936] - 	internalTcpAddr : 192.168.56.1:52032
  INFO root [2019-09-14 17:05:46,937] - 	componentID : 6000
  WARN root [2019-09-14 17:05:46,940] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 17:05:48,445] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 2560 ----
  INFO root [2019-09-14 17:05:48,445] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 17:05:48,445] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 17:05:48,446] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 17:05:48,855] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 17:05:48,856] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 17:05:55,988] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 17:05:56,002] - Components::findComponents: found logger, addr:192.168.56.1:52026
  INFO root [2019-09-14 17:05:56,003] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 17:05:56,003] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 17:05:56,004] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 17:05:56,008] - Components::findComponents: found dbmgr, addr:192.168.56.1:52028
  INFO root [2019-09-14 17:05:56,008] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 17:05:56,008] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 17:05:56,110] - Components::findComponents: found dbmgr, addr:192.168.56.1:52028
  WARN root [2019-09-14 17:05:57,111] - Forwarding logs to logger[192.168.56.1:52026/0/0/0]...
 ERROR root [2019-09-14 17:09:12,046] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=46346426086, lastReceivedTime=46289472929, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 17:09:12,048] - The following logs sent to logger failed:
  WARN root [2019-09-14 17:09:12,049] - ==> [2019-09-14 17:09:12 041] TCPPacketReceiver::processPendingEvents(192.168.56.1:52026): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:09:12,049] - ==> [2019-09-14 17:09:12 047] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=46346426086, lastReceivedTime=46289472929, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 17:09:12,154] - TCPPacketReceiver::processPendingEvents(192.168.56.1:52028): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:09:12,161] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=46346685598, lastReceivedTime=46289861733, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 17:09:12,169] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 17:09:12,171] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 17:09:12,171] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 17:09:12,172] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 17:09:33,951] - 
  INFO root [2019-09-14 17:09:33,953] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 17:09:33,954] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 17:09:33,954] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 17:09:33,954] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 17:09:33,996] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:52071 )
  INFO root [2019-09-14 17:09:33,996] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:52071, SOMAXCONN=5.
  INFO root [2019-09-14 17:09:33,996] - server-configs:
  INFO root [2019-09-14 17:09:33,997] - 	internalTcpAddr : 192.168.56.1:52071
  INFO root [2019-09-14 17:09:33,997] - 	componentID : 6000
  WARN root [2019-09-14 17:09:33,998] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 17:09:34,412] - Components::findLogger: found logger, addr:192.168.56.1:52067
  INFO root [2019-09-14 17:09:34,413] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 17:09:34,413] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 17:09:34,420] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 5100 ----
  INFO root [2019-09-14 17:09:34,420] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 17:09:34,421] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 17:09:34,421] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 17:09:34,874] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 17:09:34,890] - Components::process(): Request for the process of identity...
  WARN root [2019-09-14 17:09:41,006] - Forwarding logs to logger[192.168.56.1:52067/0/0/0]...
 ERROR root [2019-09-14 17:10:46,667] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=46559325469, lastReceivedTime=46530469290, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 17:10:46,671] - The following logs sent to logger failed:
  WARN root [2019-09-14 17:10:46,671] - ==> [2019-09-14 17:10:46 662] TCPPacketReceiver::processPendingEvents(192.168.56.1:52067): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:10:46,672] - ==> [2019-09-14 17:10:46 668] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=46559325469, lastReceivedTime=46530469290, inactivityExceptionPeriod=134887437)
  WARN root [2019-09-14 17:10:46,781] - TCPPacketReceiver::processPendingEvents(192.168.56.1:52068): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:10:46,790] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=46559602502, lastReceivedTime=46530614842, inactivityExceptionPeriod=134887437)
  INFO root [2019-09-14 17:10:46,807] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 17:10:46,812] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 17:10:46,812] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 17:10:46,813] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-14 17:11:50,621] - 
  INFO root [2019-09-14 17:11:50,649] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-14 17:11:50,650] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-14 17:11:50,651] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-14 17:11:50,651] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-14 17:11:50,687] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:52106 )
  INFO root [2019-09-14 17:11:50,687] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:52106, SOMAXCONN=5.
  INFO root [2019-09-14 17:11:50,687] - server-configs:
  INFO root [2019-09-14 17:11:50,687] - 	internalTcpAddr : 192.168.56.1:52106
  INFO root [2019-09-14 17:11:50,688] - 	componentID : 6000
  WARN root [2019-09-14 17:11:50,693] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-14 17:11:52,206] - ---- cellappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: AFC10615FCF531D416E7A56B687CCF1A. Config: Debug X86. Built: 09:54:16 May  8 2019. AppID: 6000. UID: 2082. PID: 15712 ----
  INFO root [2019-09-14 17:11:52,207] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-14 17:11:52,208] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-14 17:11:52,208] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-14 17:11:52,647] - ---- cellappmgr is running ----
 DEBUG root [2019-09-14 17:11:52,658] - Components::process(): Request for the process of identity...
  INFO root [2019-09-14 17:11:59,849] - Components::findComponents: find logger(1)...
  INFO root [2019-09-14 17:11:59,926] - Components::findComponents: found logger, addr:192.168.56.1:52104
  INFO root [2019-09-14 17:11:59,926] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 17:11:59,926] - Components::addComponent[logger], uid=2082, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 17:11:59,929] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-14 17:12:00,031] - Components::findComponents: found dbmgr, addr:192.168.56.1:52105
  INFO root [2019-09-14 17:12:00,031] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-14 17:12:00,031] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-14 17:12:00,031] - Components::findComponents: found dbmgr, addr:192.168.56.1:52105
  WARN root [2019-09-14 17:12:01,032] - Forwarding logs to logger[192.168.56.1:52104/0/0/0]...
 ERROR root [2019-09-14 17:13:09,194] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=46880007932, lastReceivedTime=46838207798, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 17:13:09,200] - The following logs sent to logger failed:
  WARN root [2019-09-14 17:13:09,200] - ==> [2019-09-14 17:13:09 190] TCPPacketReceiver::processPendingEvents(192.168.56.1:52104): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:13:09,201] - ==> [2019-09-14 17:13:09 194] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=46880007932, lastReceivedTime=46838207798, inactivityExceptionPeriod=134887564)
  WARN root [2019-09-14 17:13:09,301] - TCPPacketReceiver::processPendingEvents(192.168.56.1:52105): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-14 17:13:09,302] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=46880252413, lastReceivedTime=46838353321, inactivityExceptionPeriod=134887564)
  INFO root [2019-09-14 17:13:09,305] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-14 17:13:09,308] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-14 17:13:09,308] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-14 17:13:09,309] - ThreadPool::destroy(): starting size 2.
